app/uk/gov/hmrc/customs/declaration/model/actionbuilders/actionBuildersModel.scala

```diff
@@ -159,7 +159,7 @@ case class AuthorisedRequest[A](
   clientId: ClientId,
   authorisedAs: AuthorisedAs,
   request: Request[A],
```

Comment:
I think this should be moved to the `AuthorisedAs` subclasses as it is to do with authorisation